ts/Core/Globals.ts

```diff
@@ -201,9 +201,11 @@ namespace Globals {
         svg = !!(
             doc?.createElementNS?.(SVG_NS, 'svg') as SVGSVGElement|undefined
         )?.createSVGRect,
-        pageLang = (
-            doc?.documentElement?.closest('[lang]') as HTMLDOMElement|null
-        )?.lang,
+        // Notice: optimizing pageLang might cause ES5 build error (#22450).
+        pageLang = typeof doc?.documentElement?.closest === 'function' &&
+            (
+                doc.documentElement.closest('[lang]') as HTMLDOMElement|null
+            )?.lang,
```

Comment:
As Sophie suggests, I think we should add the polyfill in the es5-specific code instead of bogging down our main code with IE11-specific code. [Element.closest](https://caniuse.com/element-closest) has been generally supported since 2017 in modern browsers.